client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.ts

```diff
@@ -488,7 +488,7 @@ export class MotionContentComponent extends BaseMotionDetailChildComponent {
             agenda_parent_id: [],
             submitter_ids: [[]],
             supporter_user_ids: [[]],
-            workflow_id: [],
+            workflow_id: [+this.meetingSettingService.instant(`motions_default_workflow_id`)],
```

Comment:
If this form is also used for amendments, you may have to use `motions_default_amendment_workflow_id`

Reply:
`AmendmentCreateWizardComponent` is used to create amendments.

Reply:
ok, because in line 484 above this, it is checked whether this is a paragraph bsaed amendment, that's where my thought came from. Is this check unnecessary then?

Reply:
The component is also used for editing motions. But in that case a workflow is already set.

Reply:
ok!